src/com/seafile/seadroid2/ui/activity/BrowserActivity.java

```diff
@@ -360,13 +437,19 @@ public int getIconResId(int index) {
 
         @Override
         public int getCount() {
-            return ICONS.length;
+            if (!isHideActivityTab)
+                return ICONS.length;
+            else
+                return 2;
         }
     }
 
     @Override
     protected void onResume() {
         super.onResume();
+
+        fetchServerInfo();
```

Comment:
Send a server info request in `onResume()` would result in too many requests. I suggest only fetch server info once (in `onCreate`), and store the info in a static variable (for example in hash map `Map<account signature, ServerInfo>`  to hold the server info for each account), so that the server info request are only sent once.